src/styles/container.scss

```diff
@@ -1,5 +1,5 @@
 .watch-container {
-  @include square($watch-container-size);
+  @include circle($watch-container-size);
   position: relative;
   background-color: rgb(239, 237, 252);
   margin: auto;
```

Comment:
move the `margin` to the `.body__watch` class (the element of `body` block)
and add this class to your watch 😉